controllers/numaresourcesscheduler_controller.go

```diff
@@ -259,7 +259,7 @@ func configParamsFromSchedSpec(schedSpec nropv1.NUMAResourcesSchedulerSpec, cach
 	}
```

Comment:
the commit message is empty here, but this is one of the rare cases on which is OK.
For extra point you can hint (1-2 lines) which feature you want to consume which prompted the bump, but the reason why is already pretty obvious from the PR context, so minimalistic commit message is fine.